radicle-cli/src/commands/issue.rs

```diff
@@ -23,12 +23,12 @@ pub const HELP: Help = Help {
 Usage
 
     rad issue [<option>...]
-    rad issue delete <id> [<option>...]
+    rad issue delete <issue> [<option>...]
```

Comment:
renamed `id` to `issue` for better readability

Reply:
Should we use `<issue-id>`?